### PR TITLE
perf(sa): fix console.time & sa performance

### PR DESCRIPTION
**Pull request by qubaomingg**
### 👀 PR includes
🎨 Enhance
- [x] Improve the performance
### 📝 Description
- 修复console.time
- 解决渲染2次问题

### 🖼️ Screenshot

| Before                         | After                         |
| ------------------------------ | ------------------------------ |
| image | image |

### CODE REVIEW

File: packages/s2-core/src/common/interface/basic.ts [hunk 1]
```diff
@@ -210,7 +210,10 @@ export type CellCallback = (
   ...restOptions: unknown[]
 ) => S2CellType;
 
-export type DataCellCallback = (viewMeta: ViewMeta) => S2CellType;
+export type DataCellCallback = (
+  viewMeta: ViewMeta,
+  s2?: SpreadSheet,
```
Comment (xingwanying) [inline packages/s2-core/src/common/interface/basic.ts]: 嗳，为啥又加上了s2? viewMeta里能拿到嗷～

**Review by xingwanying — APPROVED** [review]
LG

File: packages/s2-core/src/common/debug/index.ts [hunk 2]
```diff
@@ -22,15 +22,10 @@ export class DebuggerUtil {
...
   public debugCallback = (info: string, callback: () => void) => {
     if (this.debug) {
-      try {
-        console.time(info);
-      } catch (e) {
-        // timer already exist
-        console.timeEnd(info);
-        callback();
-      }
+      const start = window.performance.now();
```
Comment (lijinke666) [inline packages/s2-core/src/common/debug/index.ts:25]: 为什么要加  window 前缀

Comment (wjgogogo) [inline packages/s2-core/src/common/debug/index.ts:25]: 可以去掉 window前缀

Comment (qubaomingg) [reply packages/s2-core/src/common/debug/index.ts:25]: 去掉

Comment (qubaomingg) [reply packages/s2-core/src/common/debug/index.ts:25]: 去掉

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**